python/ccf/cose.py

```diff
@@ -173,8 +228,34 @@ def _parser():
     return parser
 
 
+def _sign_parser():
+    parser = _common_parser(_SIGN_DESCRIPTION)
+    parser.add_argument(
+        "--signing-key",
+        help="Path to signing key, PEM-encoded",
+        type=str,
+        required=True,
+    )
+    return parser
+
+
+def _finish_parser():
+    parser = _common_parser(_FINISH_DESCRIPTION)
+    parser.add_argument(
+        "--signature",
+        help='Path to JSON file with a "value" field containing a JWS',
```

Comment:
Doesn't contain a JWS, but rather a raw base64 encoded signature. It's important to know whether this is assuming DER-style or IEEE for EC signatures.

Reply:
Yes, good point. Unfortunately the AKV doc doesn't say very much (https://learn.microsoft.com/en-us/rest/api/keyvault/keys/sign/sign?source=recommendations&tabs=HTTP), but it does seem to be raw.